server.go

```diff
@@ -17,14 +17,24 @@ import (
 )
 
 type OpenAPIConfig struct {
-	DisableSwagger   bool                              // If true, the server will not serve the Swagger UI nor the OpenAPI JSON spec
-	DisableSwaggerUI bool                              // If true, the server will not serve the Swagger UI
-	DisableLocalSave bool                              // If true, the server will not save the OpenAPI JSON spec locally
-	SwaggerUrl       string                            // URL to serve the swagger UI
-	UIHandler        func(specURL string) http.Handler // Handler to serve the OpenAPI UI from spec URL
-	JsonUrl          string                            // URL to serve the OpenAPI JSON spec
-	JsonFilePath     string                            // Local path to save the OpenAPI JSON spec
-	PrettyFormatJson bool                              // Pretty prints the OpenAPI spec with proper JSON indentation
+	// Overrides the server description generated by fuego
+	Server *openapi3.Server
+	// If true, the server will not serve the Swagger UI nor the OpenAPI JSON spec
+	DisableSwagger bool
```

Comment:
Not sure that a chunk (`*openapi3.Server`) of the OpenAPI Description should be in the `OpenAPIConfig`.

`OpenAPIConfig` is more about "How will Fuego handle the output of the OpenAPI Description". But as there's no comment, you wouldn't have guessed it, sorry about that!

Reply:
Yeah I figured as much. I personally think this does have to do with the OpenApi description. That's all this change is really focusing on.